.travis.yml

```diff
@@ -1,10 +1,45 @@
-language: python
-python: 2.7
-env:
-  - TOXENV=flake8
-install:
-  - pip install tox
-script:
-  - tox
-notifications:
-  email: kpennington@mozilla.com
+cache: pip
+
+sudo: required
+jobs:
+  include:
+    - stage:
+      language: python
+      python: 3.6
+      env: TOXENV=py27
+      before_install:
+        - wget -O /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.20.0/geckodriver-v0.20.0-linux64.tar.gz
+        - mkdir $HOME/geckodriver && tar xvf /tmp/geckodriver.tar.gz -C $HOME/geckodriver
+        - export PATH=$HOME/geckodriver:$PATH
+        - firefox --version
+        - geckodriver --version
```

Comment:
It looks like you aren't starting any kind of window manager. If you don't want these to run headless, you need to start a window management service. See [this](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI).

Or you could run firefox as headless using the env variable ```MOZ_HEADLESS=1```

Reply:
I did try setting `MOZ_HEADLESS=1`, and I'm still seeing the same error.

As far as checking for `not None`, I'm not really sure what to check for, unless I save JSON data and check the new data against that.

I really just want to see if it returns anything.

What do you suggest?

Reply:
You could create a simple webpage with your own properties and such and then check against that. That way you know what is on the page and the test should return that each time.

Reply:
I think you could either have firefox open a page within a directory or you could use [this](https://docs.python.org/2/library/simplehttpserver.html).